src/post.proj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
+
+  <!-- provides a hook for executing targets after the product and tests have been built -->
+  <Target Name="Build" DependsOnTargets="$(PostBuildDependsOn)" />
+
+  <Import Project="$(ToolsDir)Build.Post.targets" Condition="Exists('$(ToolsDir)Build.Post.targets')" />
```

Comment:
I see what you're getting at.  AfterTargets would work as well and simplifies hooking them up.  I'll make this change.